contents/handbook/engineering/incidents.md

```diff
@@ -35,46 +35,65 @@ Please refer to the following guidance when choosing the severity for your incid
 
-
-## What happens during an incident
+## What happens during an incident?
 
 The person who raised the incident is the incident lead. It’s their responsibility to:
-- Make sure the right people join the zoom. This includes [the current on call person](https://posthog.pagerduty.com/service-directory/P43Y0E8). Optionally, add people from Infra and [the feature owner](https://posthog.com/handbook/engineering/feature-ownership) if relevant. Also ping Tim and James G so they're aware.
-- take notes in the incident channel. This should include time stamps, and is a brain dump of everything that we know, and everything that we are or have tried. This will give us much more opportunity to learn from the incident afterwards.
-- Update the [status banner on app](https://app.posthog.com/feature_flags/984)
+- Make sure the right people join the Zoom. This includes [the current on call person](https://posthog.pagerduty.com/service-directory/P43Y0E8). Optionally, add people from Infra and [the feature owner](https://posthog.com/handbook/engineering/feature-ownership) and Marketing if relevant. Ping Tim and James G so they're aware. Marketing can assist on running communication if required.
+- Take notes in the incident channel. This should include time stamps, and is a brain dump of everything that we know, and everything that we are or have tried. This will give us much more of an opportunity to learn from the incident afterwards.
+- Update the [status banner on app](https://app.posthog.com/feature_flags/984). There are some templates below to make this easier.
 - Update the [status page](https://uptimerobot.com/statuspage.php)
 - Update users in [#announcements](https://posthogusers.slack.com/archives/CT7HXDEG3)
 
 If the person who raised the incident is the best person to debug the issue, they should hand over the incident lead role to someone else on the call.
 
 [You can find all of our production runbooks + specific strategies for debugging outages here (internal)](https://github.com/PostHog/product-internal/tree/main/infrastructure/runbooks)
 
+### Customer communications
+
+The main way to communicate an incident to customers is via [the banner feature flag](https://app.posthog.com/feature_flags/984). It's the responsibility of the incident lead to enable the banner, and to disable it when the incident is resolved.
+
+All in-app banners should link to a resource offering more information, usually the status page.
+
+The banner should simply state the user impact and direct users to more detailed information. Keep it simple, and direct.
+
+Example flag payloads:
+`Events from the last 5 days may be duplicated due to an error. [More info](https://status.posthog.com/).`
+`Event ingestion is currently delayed by three hours. [More info](https://status.posthog.com/).`
+
+If in doubt, a generic message can suffice:
+
+`We're investigating an on-going incident. [Check our status page for updates.](https://status.posthog.com/)`
```

Comment:
In this case folks should just remove the message and have the default message be displayed. 
https://github.com/PostHog/posthog/blob/c681b422a39e1305be49688138391f65c722590b/frontend/src/layout/navigation/TopBar/announcementLogic.ts#L21 <- should we update that?